cmd/install/assets/hypershift-operator/hypershift.openshift.io_hostedclusters.yaml

```diff
@@ -7550,38 +7547,44 @@ spec:
                         default: default
                         description: |-
                           ResourceGroupName is the name of an existing resource group where all cloud resources created by the Hosted
-                          Cluster are to be placed.
+                          Cluster are to be placed. The resource group is expected to exist under the same subscription as SubscriptionID.
+
+
                           In ARO HCP, this will be the managed resource group where customer cloud resources will be created.
 
 
+                          Resource group naming requirements can be found here: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.ResourceGroup.Name/.
+
+
                           Example: if your resource group ID is /subscriptions/<subscriptionID>/resourceGroups/<resourceGroupName>, your
                                    ResourceGroupName is <resourceGroupName>.
+                        pattern: ^[a-zA-Z0-9_()\-\.]{1,89}[a-zA-Z0-9_()\-]$
                         type: string
                         x-kubernetes-validations:
                         - message: ResourceGroupName is immutable
                           rule: self == oldSelf
-                      securityGroupName:
+                      securityGroupID:
                         description: |-
-                          SecurityGroupName is the name of an existing security group on SubnetName. This field is provided as part of the
-                          configuration for the Azure cloud provider, aka Azure cloud controller manager (CCM).
-
-
-                          Example: if your Network Security Group ID is /subscriptions/<subscriptionID>/resourcegroups/<resourceGroupName>/providers/Microsoft.Network/networkSecurityGroups/<securityGroupName>,
-                                   your SecurityGroupName is <securityGroupName>
+                          SecurityGroupID is the ID of an existing security group on the SubnetID. This field is provided as part of the
+                          configuration for the Azure cloud provider, aka Azure cloud controller manager (CCM). This security group is
+                          expected to exist under the same subscription as SubscriptionID and in the same resource group VnetID is located
```

Comment:
>and in the same resource group VnetID is located
same comment as above

Reply:
That's the current assumption atm until we make changes in [HOSTEDCP-1557](https://issues.redhat.com/browse/HOSTEDCP-1557).